extensions/hibernate-search-orm-elasticsearch/deployment/src/main/resources/dev-templates/embedded.html

```diff
@@ -0,0 +1,4 @@
+<a href="{urlbase}/entities" class="badge badge-light">
+    <i class="fa fa-eye fa-fw"></i>
```

Comment:
Small detail but why not use `fa-search` for the icon?

Reply:
Indeed, it looks better.